tools/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -422,7 +430,7 @@ class QualificationAppInfo(
 
       val estimatedInfo = QualificationAppInfo.calculateEstimatedInfoSummary(estimatedGPURatio,
         sparkSQLDFWallClockDuration, appDuration, taskSpeedupFactor, appName, appId,
-        sqlIdsWithFailures.nonEmpty, unSupportedExecs, unSupportedExprs)
+        sqlIdsWithFailures.nonEmpty, unSupportedExecs, unSupportedExprs, clusterId, jobId, runName)
```

Comment:
similar here can we make this a map of extra confs?

Reply:
if a Map doesn't work for some reason then make it a case class so we don't have to keeping adding individual parameters

Reply:
Done